bindings/wallet-cordova/src/android/WalletPlugin.java

```diff
@@ -127,6 +130,17 @@ private void walletTotalFunds(JSONArray args, CallbackContext callbackContext) t
         }
     }
 
+    private void walletId(JSONArray args, CallbackContext callbackContext) throws JSONException {
+        Long walletPtr = args.getLong(0);
+
+        try {
+            byte[] id = Wallet.id(walletPtr);
+            // TODO
```

Comment:
@NicolasDP , I just wanted to delete my comment but it deleted the whole thread. Sorry :/

Reply:
I found out the other day that cordova automagically encodes byte[] as base64 (it has an overload for `success`), and does the inverse on js. So we could just do that and either hexencode the `ArrayBuffer` in the plugin's js code or just leave it to the app I think.